### DISCUSSION TITLE
Best technique for marking/highlighting clues?

### DISCUSSION THREAD

**Discussion by DKMiller71** [viresh-ratnakar/exolve#98] [Q&A] [answered]
Pondering how best to set it up such that certain clues are marked / highlighted / bolded... any thoughts?  Ideas are a) mark them with asterisk, b) wrap the clue in a span element to make it bolded, c) put them in a different color (probably not accessibility or print friendly), other?

The idea is that when they've solved the puzzle, they'll have to do something special with the marked answers to get a meta answer.

**Comment by viresh-ratnakar** [comment, marked as answer]
Sorry, I missed this entirely (I do not get notifications for new items in "Discussions" .. perhaps there's a way for me to fix that?).

Prefixing the clues with a * or &dagger; or &ddagger; or some other distinctive unicode character seems like the simplest solution to me.
